BoxContentSDK/BoxContentSDK/OAuth2/BOXAuthorizationViewController.m

```diff
@@ -189,13 +189,21 @@ - (void)viewWillDisappear:(BOOL)animated
     [super viewWillDisappear:animated];
 }
 
+- (UIApplication *) box_sharedApplication {
+    UIApplication *app = nil;
+    if ([[UIApplication class] respondsToSelector:@selector(sharedApplication)]) {
+        app = [[UIApplication class] performSelector:@selector(sharedApplication)];
+    }
+    return app;
+}
+
```

Comment:
```suggestion
```

Reply:
You can actually remove this now, since `box_sharedApplication` already exists, right?
https://github.com/box/box-ios-sdk/blob/objective-c-maintenance/BoxContentSDK/BoxContentSDK/Categories/UIApplication%2BExtensionSafeAdditions.m#L16

Reply:
done